vignettes/rgl.Rmd

```diff
@@ -230,6 +230,37 @@ provide low-level support for `r linkfn("setAxisCallbacks")`.
 
 ## Controlling the Look of the Scene
 
+### Camera angle
+
+By default when you open a new plot:
+
+* The x axis goes from left to right.
+* The y axis goes from near the camera to far away.
+* The z axis goes from down to up.
+
+You can change the camera angle simply by dragging the picture with
+the mouse.
+
+To set the camera angle programmatically, use `r indexfns("view3d")`. 
+This uses polar coordinates:
+
+* `theta` sets the rotation around the z axis, in degrees. 
+* `phi` sets the "vertical" rotation around the x axis, from -90 to 90
+  degrees.
+  
+The default angle is roughly `theta = 0, phi = -60`. Starting from
```

Comment:
It's actually more like `theta = 0, phi = -70`.  The reason for the big `phi` value is the difference between OpenGL coordinates and the coordinates that seem natural to me (having z point up).  But I don't know if it would help to explain this, or just confuse things more.

Reply:
Fixed. I think trying to explain OpenGL isn't going to help new users. I've managed to use rgl so far without learning about it.